types/marko/express.d.ts

```diff
@@ -1,20 +1,10 @@
 import * as express from 'express';
 import Template from './src/runtime/html/Template';
-import { IRouterMatcher as ExpressRouterMatcher, NextFunction, PathParams } from 'express-serve-static-core';
 
-declare function m(): m.Application;
-
-declare namespace m {
-    interface Response extends express.Response {
+declare module 'express-serve-static-core' {
+    interface Response {
         marko(template: Template, data?: any): void;
     }
-
-    interface Application extends express.Application {
-        get: ((name: string) => any) & ExpressRouterMatcher<this> & MarkoRouterMatcher;
-    }
-
-    type MarkoRouterMatcher = (path: PathParams, ...handlers: RequestHandler[]) => Application;
-    type RequestHandler = (req: express.Request, res: Response, next?: NextFunction) => any;
 }
 
-export = m;
+export = express;
```

Comment:
Is this line needed? As far as I understand, this definition is a module augmentation on `express-serve-static-core` and no functions/values are exported directly.

Reply:
According to the (Marko.js docs)[https://markojs.com/docs/express/] we need to do this:
```
var express = require("express");
var markoExpress = require("marko/express")

var app = express();
app.use(markoExpress()); // <- this is the reason I export `express`
```

I have this example in the tests and I don't know how to make it better, do you have any ideas?

Reply:
Oops, I didn't read at the docs carefully enough. Your change looks correct.